File: Algorithms/027.remove-element/solution.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    let arr = []
    for (let i=0; i<nums.length; i++) {
        if (nums[i] !== val) arr.push(nums[i])
    }
    console.log(arr)
    return arr.length
};

console.log(removeElement([3, 2,2,3,3], 3))

File: Algorithms/027.remove-element/solution.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        arr = []
        for i in nums:
            if i != val:
                arr.append(i)
        return len(arr)


print(Solution().removeElement([2, 2, 3, 3, 2], 3))
